testdata/well_tested/src/numbers.rs

```diff
@@ -6,6 +6,14 @@ fn is_double(a: u32, b: u32) -> bool {
     a == 2 * b
 }
 
+fn negate_int(a: i32) -> i32 {
```

Comment:
These might be better with s/int/i32/ in the same, or just make them parameterized by some type and then you can test i32, u32, f32, etc.

Reply:
Sure, I guess I can make it generic over `T: std::ops::Neg`.

Reply:
I guess either is OK, but, in this case we're really focusing on negation/inversion. The other functions I think will make sure we try those constant values?

Reply:
True, negation/inversion is the focus.

> The other functions I think will make sure we try those constant values?

I'm not sure that I understand. Maybe it doesn't matter, if you don't mind what I've done now.

Reply:
What I meant was, there should already be other test cases that exercise replacement of `fn a() -> usize { ... }` with `{ 0 }` etc. It's not important that they're produced here. However, yes, your current code is fine as it is.